deepchem/data/data_loader.py

```diff
@@ -1745,7 +1770,11 @@ def _featurize_shard(self, shard):
             )
         if 'weight' in shard.keys():
             weight = shard['weight']
-            x = DFTEntry.create(e_type, true_val, systems, weight)
+            x = DFTEntry.create(e_type,
```

Comment:
A DFTEntry object should not know about the device. That is a feature of the model and not of the data